Buildings/Fluid/Dehumidifiers/Desiccant/ElectricCoilSpeedControlled.mo

```diff
@@ -0,0 +1,170 @@
+within Buildings.Fluid.Dehumidifiers.Desiccant;
+model ElectricCoilSpeedControlled
+  "Desiccant dehumidifier with an electric heating coil and a variable speed wheel"
+  extends BaseClasses.PartialDesiccant(
+    vol(nPorts=2));
+  parameter Real etaHea(
+    final min=0,
+    final max=1)
+    "Heater efficiency"
+    annotation (Dialog(group="Efficiency"));
+  parameter Real c[:] = {1}
+    "Coefficients for power consumption curve for the rotor, 
+     P/P_nominal = sum a_i uSpe^(i-1). The sum(a) of the elements must be equal to 1"
```

Comment:
don't include any equation in comment. Also don't include the variable names in the comment.

Reply:
Removed the equation and variable name.